gnomad_qc/v4/sample_qc/relatedness.py

```diff
@@ -540,6 +663,97 @@ def main(args):
     )
+    finalize.add_argument(
+        "--parent-child-max-ibd0-or-ibs0-over-ibs2",
+        help=(
+            "Maximum value of IBD0 (if '--finalize-relatedness-method' is 'pc_relate') "
+            "or IBS0/IBS2 (if '--finalize-relatedness-method' is 'cuking') for a "
+            "parent-child pair."
+        ),
+        default=5.2e-5,
+        type=float,
+    )
+    finalize.add_argument(
+        "--second-degree-sibling-lower-cutoff-slope",
+        help=(
+            "Slope of the line to use as a lower cutoff for second degree relatives "
+            "and siblings from parent-child pairs."
+        ),
+        default=-1.9e-3,
+        type=float,
+    )
+    finalize.add_argument(
+        "--second-degree-sibling-lower-cutoff-intercept",
+        help=(
+            "Intercept of the line to use as a lower cutoff for second degree "
+            "relatives and siblings from parent-child pairs."
+        ),
+        default=5.8e-4,
+        type=float,
+    )
+    finalize.add_argument(
+        "--second-degree-upper-sibling-lower-cutoff-slope",
+        help=(
+            "Slope of the line to use as an upper cutoff for second degree relatives "
+            "and a lower cutoff for siblings."
+        ),
+        default=-1e-2,
+        type=float,
+    )
+    finalize.add_argument(
+        "--second-degree-upper-sibling-lower-cutoff-intercept",
+        help=(
+            "Intercept of the line to use as an upper cutoff for second degree "
+            "relatives and a lower cutoff for siblings."
+        ),
+        default=2.2e-3,
+        type=float,
+    )
+    finalize.add_argument(
+        "--duplicate-twin-min-kin",
+        help="Minimum kinship for duplicate or twin pairs.",
+        default=0.42,
+        type=float,
+    )
+    finalize.add_argument(
+        "--duplicate-twin-ibd1-min",
+        help=(
+            "Minimum IBD1 cutoff for duplicate or twin pairs. Only used when "
+            "'--finalize-relatedness-method' is 'pc_relate'. Note: the min is used "
+            "because pc_relate can output large negative values in some corner cases."
+        ),
+        default=-0.15,
+        type=float,
+    )
+    finalize.add_argument(
+        "--duplicate-twin-ibd1-max",
+        help=(
+            "Maximum IBD1 cutoff for duplicate or twin pairs. Only used when "
+            "'--finalize-relatedness-method' is 'pc_relate'."
+        ),
+        default=0.1,
+        type=float,
+    )
+
+    drop_related_samples = parser.add_argument_group(
+        "Compute related samples to drop",
+        "Arguments used to determine related samples that should be dropped from "
+        "the ancestry PCA or release.",
+    )
+    drop_related_samples.add_argument(
+        "--compute-related-samples-to-drop",
+        help="Determine the minimal set of related samples to prune for ancestry PCA.",
+        action="store_true",
+    )
+    drop_related_samples.add_argument(
+        "--release",
```

Comment:
If the above suggestion is accepted then 
```suggestion
        "--compute-related-samples-to-drop-for-release",
```
I  think this can be reassigned to release in the script but from a command line help perspective, it helps show these are doing similar thing.